src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -76,8 +79,11 @@
 @media (--tablet-and-down) {
   .gridParent {
     display: block;
-    padding-bottom: 80px;
-    padding-top: 40px;
+    padding: 50px 0 80px;
+  }
+
+  .gridParent.v2 {
+    padding: 40px 0 80px;
```

Comment:
I think this isn't actually necessary, since it looks like both versions have 50px top padding. This may have very subtly changed in the designs between when we first implemented the component and now.